client/state/signup/progress/actions.js

```diff
@@ -8,6 +8,7 @@ import { includes, isEmpty, reduce, snakeCase, toPairs } from 'lodash';
  */
 import {
 	SIGNUP_PROGRESS_SAVE_STEP,
+	SIGNUP_PROGRESS_SET_RESUME_AFTER_LOGIN,
```

Comment:
A nitpick with the action naming here:
`SET` would ideally be last. If we were to have other similar actions, such as for `CLEAR`ing `the state item, we would want that to be next to each other alphabetically:

```js
// out of order:
const SIGNUP_PROGRESS_CLEAR_RESUME_AFTER_LOGIN = ...
const SIGNUP_PROGRESS_RESET = ...
const SIGNUP_PROGRESS_SET_RESUME_AFTER_LOGIN = ...

// in order:
const SIGNUP_PROGRESS_RESET = ...
const SIGNUP_PROGRESS_RESUME_AFTER_LOGIN_CLEAR = ...
const SIGNUP_PROGRESS_RESUME_AFTER_LOGIN_SET = ...
```

Reply:
Good call! I'll do some renaming.

Reply:
Updated!